crates/cli/src/lib.rs

```diff
@@ -683,13 +704,14 @@ fn print_problems(problems: Problems, total_time: std::time::Duration) {
 }
 
 fn roc_run<'a, I: IntoIterator<Item = &'a OsStr>>(
-    arena: Bump, // This should be passed an owned value, not a reference, so we can usefully mem::forget it!
+    arena: &Bump, // This should be passed an owned value, not a reference, so we can usefully mem::forget it!
```

Comment:
right so this is no longer true, because the `layout_interner` stores stuff into this `Arena`. I also think it's irrelevant, because `execve` just ends the process; I don't think `drop` runs before that.